File: KeepIt/app/src/main/java/com/utbm/KeepIt/MainActivity.java
package com.utbm.KeepIt;

import androidx.appcompat.app.AppCompatActivity;

import android.database.Cursor;
import android.os.Bundle;
import android.widget.TextView;

import DAO.LoginDao;

public class MainActivity extends AppCompatActivity {
    private LoginDao userDao;
    private int age;
    private int height;
    private float weight;
    private int level;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //绑定对应的界面
        setContentView(R.layout.activity_main);

        System.out.println("------------------------------------------------------------------------------------------------");
        userDao = new LoginDao(this);

        String name = getIntent().getStringExtra("username");
        Cursor userInfo = userDao.getUserInfoByName(name);
        userInfo.moveToFirst();

            this.age = userInfo.getInt(userInfo.getColumnIndex("age"));
            this.height = userInfo.getInt(userInfo.getColumnIndex("height"));
            this.weight = userInfo.getFloat(userInfo.getColumnIndex("weight"));
            this.level = userInfo.getInt(userInfo.getColumnIndex("level"));

            TextView ageView = findViewById(R.id.userAge);
            TextView heightView = findViewById(R.id.userHeight);
            TextView weightView = findViewById(R.id.userWeight);
            TextView levelView = findViewById(R.id.userLevel);

            ageView.setText(""+this.age+" "+this.height+" "+this.weight+" "+ this.level);


    }



}

File: LO52_2019_Equal/TPs/TP1_AP/app/src/main/java/com/equal/tp1_hello_world_ap/MainActivity.kt
package com.equal.tp1_hello_world_ap

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main. activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        buttonToHelloWorld.setOnClickListener(){
            val intent = Intent(this, HelloActivity::class.java)
            startActivity(intent)
        }
    }
}
